components/BottomMenu.js

```diff
@@ -13,10 +15,12 @@ function BottomMenu () {
     const [selectedBuilding, setSelectedBuilding] = React.useState("");
     const [iconSelected, setIconSelected] = React.useState(false);
     const [switchVal, setSwitchVal] = React.useState(true);
+    const [getInside, setGetInside] = React.useState(false);
 
     CurrentLocation();
 
     AsyncStorage.setItem("toggle", switchVal.toString());
+    AsyncStorage.setItem("getInsideBuilding", getInside.toString());
 
     const buildingSelected = async () => {
```

Comment:
Can we use a more descriptive function name? It's confusing because the constant "selectedBuilding" has an almost identical name and it gets confusing.

Reply:
You are comparing `selectedBuilding` name with which const ?

Reply:
with buildingSelected

Reply:
the function **buildingSelected** and the constant selectedBuilding have an almost identical name and it's not too easy to differentiate between the two :)

Reply:
> Some of the room numbers are not inside the boxes starting from floor 10 I believe :/. @csbduzi has the same issue.
> 
> ![image](https://user-images.githubusercontent.com/35609559/76273107-6b496080-6253-11ea-9a4f-2be785edb3c3.png)



> Some of the room numbers are not inside the boxes starting from floor 10 I believe :/. @csbduzi has the same issue.
> 
> ![image](https://user-images.githubusercontent.com/35609559/76273107-6b496080-6253-11ea-9a4f-2be785edb3c3.png)



> Some of the room numbers are not inside the boxes starting from floor 10 I believe :/. @csbduzi has the same issue.
> 
> ![image](https://user-images.githubusercontent.com/35609559/76273107-6b496080-6253-11ea-9a4f-2be785edb3c3.png)

Minor issue to add here, maybe if we can resize the room numbers to ensure that they all fit in their respective place it would make the UI nicer.